src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+
+        logger.info(
+            f"""{termColors.OKGREEN}
+
+        ============= FINISHED ===============
+        Created {total_new_entries} transition domain entries,
+        updated {total_updated_domain_entries} transition domain entries
+        {termColors.ENDC}
+        """
+        )
+
+        # Print a summary of findings (duplicate entries,
+        # missing data..etc.)
+        self.print_summary_duplications(
+            duplicate_domain_user_combos, duplicate_domains, users_without_email
+        )
+        self.print_summary_status_findings(domains_without_status, outlier_statuses)
+
+    def print_debug_mode_statements(self, debug_on, debug_max_entries_to_parse):
+        if debug_on:
+            logger.info(
+                f"""{termColors.OKCYAN}
+            ----------DEBUG MODE ON----------
+            Detailed print statements activated.
+            {termColors.ENDC}
+            """
+            )
+        if debug_max_entries_to_parse > 0:
+            logger.info(
+                f"""{termColors.OKCYAN}
+            ----------LIMITER ON----------
+            Parsing of entries will be limited to
+            {debug_max_entries_to_parse} lines per file.")
+            Detailed print statements activated.
+            {termColors.ENDC}
+            """
+            )
+
+    def get_domain_user_dict(self, domain_statuses_filename, sep):
+        """Creates a mapping of domain name -> status"""
+        # TODO: figure out latest status
+        domain_status_dictionary = defaultdict(str)
+        # NOTE: how to determine "most recent" status?
+        logger.info("Reading domain statuses data file %s", domain_statuses_filename)
+        with open(domain_statuses_filename, "r") as domain_statuses_file:  # noqa
+            for row in csv.reader(domain_statuses_file, delimiter=sep):
+                domainName = row[0].lower()
+                domainStatus = row[1].lower()
+                # print("adding "+domainName+", "+domainStatus)
+                domain_status_dictionary[domainName] = domainStatus
+        logger.info("Loaded statuses for %d domains", len(domain_status_dictionary))
+        return domain_status_dictionary
+
+    def get_user_emails_dict(self, contacts_filename, sep):
+        """Creates mapping of userId -> emails"""
+        # NOTE: is this one to many??
+        user_emails_dictionary = defaultdict(list)
+        logger.info("Reading domain-contacts data file %s", contacts_filename)
+        with open(contacts_filename, "r") as contacts_file:
+            for row in csv.reader(contacts_file, delimiter=sep):
+                userId = row[0]
+                user_email = row[6]
+                user_emails_dictionary[userId].append(user_email)
+        logger.info("Loaded emails for %d users", len(user_emails_dictionary))
+        return user_emails_dictionary
+
+    def get_mapped_status(self, status_to_map):
+        # Map statuses as follows;
```

Comment:
**comment**: This needs a docstring and it should include this comment's information

Reply:
Ideally a type hint as well.

Reply:
Done